internal/databases/sqlserver/sqlserver.go

```diff
@@ -378,10 +378,16 @@ func getLogsSinceBackup(folder storage.Folder, backupName string, stopAt time.Ti
 	return logNames, nil
 }
 
-func runParallel(f func(int) error, cnt int) error {
+func runParallel(f func(int) error, cnt int, concurency int) error {
+	if concurency == 0 {
+		concurency = cnt
+	}
+	sem := make(chan struct{}, concurency)
```

Comment:
semaphore.NewWeighted?

Reply:
chans are idiomatic for simple semaphores, and we don't need weights here